planner/languages/elixir/elixir_planner.go

```diff
@@ -3,10 +3,47 @@
 
 package elixir
 
-import "go.jetpack.io/devbox/planner/plansdk"
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"go.jetpack.io/devbox/planner/plansdk"
+)
 
 type Planner struct{}
 
+var NoElixirVersionSetErr = errors.New("No version set in mix.exs")
+
+var versionMap = map[string]string{
+	"1.9":  "elixir_1_9",
+	"1.10": "elixir_1_10",
+	"1.11": "elixir_1_11",
+	"1.12": "elixir_1_12",
+	"1.13": "elixir",
+	"1.14": "elixir_1_14",
+}
+
+type ElixirProject struct {
```

Comment:
does this need to be visible outside this `elixir` package? if not, lets rename to `elixirProject` to keep it private.

Reply:
done